### DISCUSSION TITLE
How to remove Lively

### DISCUSSION THREAD

**Discussion by txnguu** [rocksdanister/lively#251] [Q&A] [answered]
i want to go back to a normal wallpaper (not animated wallpaper) but i don't know how to do it, i deleted lively wallpaper thinking that the wallpaper would go away but it doesn't, and if i try to download lively wallpaper to delete the wallpaper it tells me it is in use the application. help, please i want to remove this background and return to normal.

**Comment by rocksdanister** [comment, marked as answer]
**Microsoft store version**
Microsoft store version of Lively is using latest MSIX packaging technology from Microsoft, it will not show up in the OLD installed app page.

Open the new "Add or Remove Program" dialog or simply uninstall straight from the startmenu right-click menu:
![image](https://user-images.githubusercontent.com/17554161/223108593-8806ae90-f176-42cd-8459-713451e26987.png)

**Installer version**
Run the uninstall from control panel or program install folder.

Further read:
https://www.pcmag.com/how-to/how-to-uninstall-programs-in-windows-10

**Comment by rocksdanister** [comment]
I don't know what you mean by delete? the uninstaller will kill any running Lively process and remove all the installed files.

**Reply by txnguu** [reply]
oh ok, thanks
